doc/whatsnew/fragments/8073.internal

```diff
@@ -0,0 +1,3 @@
+Use string-pattern-matching as a precondition for ``astroid`` inference in ``pointless-exception-statement``.
+
+Refs #8073
```

Comment:
The check isn't released yet, so I don't think we necessarily need a changelog entry for this PR. @Pierre-Sassoulas What do you think?

Reply:
Yeap, a changelog would actually be confusing.